tests/services/test_rest_svc.py

```diff
@@ -22,17 +22,17 @@ def setup_rest_svc_test(loop, data_svc):
                                                 'encryption_key': 'ADMIN123',
                                                 'exfil_dir': '/tmp'})
     loop.run_until_complete(data_svc.store(
-        Ability(ability_id='123', executors=[
+        Ability(ability_id='123', name='testA', executors=[
             Executor(name='psh', platform='windows', command='curl #{app.contact.http}')
         ])
     ))
     loop.run_until_complete(data_svc.store(
-        Ability(ability_id='456', executors=[
+        Ability(ability_id='456', name='testB', executors=[
             Executor(name='sh', platform='linux', command='whoami')
         ])
     ))
     loop.run_until_complete(data_svc.store(
-        Ability(ability_id='789', executors=[
+        Ability(ability_id='789', name='testC', executors=[
```

Comment:
So how are these new lines here being tested in the single `test_delete_operation` test below?

Reply:
Tangentially, but this fix is actually for the deadman tests. Since we no long allow name collisions for abilities, those tests were failing, since on the first ability loaded would have the "default" name. If we specify the names manually, this naming collision no longer seems to occur.

Reply:
Gotcha.